server/lib/notifications.ts

```diff
@@ -367,6 +367,59 @@ async function notifyByEmail(activity: Activity) {
   debug('notifyByEmail', activity.type);
   let collective, conversation;
   switch (activity.type) {
+    case ActivityTypes.USER_NEW_TOKEN:
+      notifyUserId(activity.UserId, activity, { sendEvenIfNotProduction: true });
+      break;
+
+    case ActivityTypes.USER_CHANGE_EMAIL:
+      notifyUserId(activity.UserId, activity, {
+        to: activity.data.emailWaitingForValidation,
+        sendEvenIfNotProduction: true,
+      });
+      break;
+
+    case ActivityTypes.ORDER_PROCESSING:
+      notifyUserId(activity.UserId, activity, {
+        from: `${activity.data.collective.name} <no-reply@${activity.data.collective.slug}.opencollective.com>`,
+      });
+      break;
```

Comment:
What if it's coming from an Organization, only the triggering user get it?

Reply:
Yes, for now, I'll keep the same behavior. We can discuss this further.